msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.CSharp.targets

```diff
@@ -28,6 +28,11 @@ Copyright (C) 2014 Xamarin Inc. All rights reserved.
 	<Import Project="Xamarin.Mac.ObjCBinding.Common.targets" />
 
 	<PropertyGroup>
+		<IsXBuild Condition="'$(MSBuildRuntimeVersion)' == ''">true</IsXBuild>
```

Comment:
It does in theory. I did not look into getting it working with msbuild, but let me see how bad it is...